demisto_sdk/commands/create_artifacts/content_artifacts_creator.py

```diff
@@ -696,12 +930,78 @@ def report_artifacts_paths(artifact_manager: ArtifactsManager):
     else:
         template = "\n\t - {}"
 
-    if artifact_manager.only_content_packs:
-        logger.info(template.format(artifact_manager.content_packs_path))
-
     logger.info(template.format(artifact_manager.content_packs_path))
 
     if not artifact_manager.only_content_packs:
         for artifact_dir in [artifact_manager.content_test_path, artifact_manager.content_new_path,
                              artifact_manager.content_all_path]:
             logger.info(template.format(artifact_dir))
+
+    if artifact_manager.zip_artifacts:
+        logger.info(f'\n\t - {artifact_manager.content_uploadable_zips_path}')
+
+
+###############################
+# Pack signing and encryption #
+###############################
+
+
+def sign_packs(artifact_manager: ArtifactsManager):
+    """Sign packs directories"""
+    if artifact_manager.signDirectory and artifact_manager.signature_key:
+        with open('keyfile', 'wb') as keyfile:
+            keyfile.write(artifact_manager.signature_key.encode())
+
+        with ProcessPoolHandler(artifact_manager) as pool:
+            futures: List[ProcessFuture] = []
+            if 'all' in artifact_manager.pack_names:
+                for pack_name, pack in artifact_manager.content.packs.items():
+                    dumped_pack_dir = os.path.join(artifact_manager.content_packs_path, pack.id)
+                    futures.append(pool.schedule(pack.sign_pack, args=(logger, dumped_pack_dir,
+                                                                       artifact_manager.signDirectory,
+                                                                       )))
+            else:
+                for pack_name in artifact_manager.pack_names:
+                    if pack_name in artifact_manager.content.packs:
+                        pack = artifact_manager.content.packs[pack_name]
+                        dumped_pack_dir = os.path.join(artifact_manager.content_packs_path, pack.id)
+                        futures.append(pool.schedule(pack.sign_pack, args=(logger, dumped_pack_dir,
+                                                                           artifact_manager.signDirectory,
+                                                                           )))
+        wait_futures_complete(futures, artifact_manager)
+        os.remove('keyfile')
```

Comment:
should probably be in a finally clause. To make sure this is deleted even if there is an exception.